docs/source/getting_started/changelog.md

```diff
@@ -2,6 +2,10 @@
 
 A summary of new feature highlights is located on the [GitHub release page](https://github.com/elyra-ai/elyra/releases).
 
+## Release 3.15.1 - 03/28/2023
```

Comment:
milestone or release target when looking at elyra-ai:main branch should be 4.0, should it not? @lresende what do you think?

Reply:
Should I change it or leave it blank so it can be filled when the next release is created?